src/njoy21/input/LEAPR/Card20/Comment.hpp

```diff
@@ -0,0 +1,18 @@
+struct Comment {
+  using Value_t = optional<std::string>;
+  static std::string name(){ return "comment"; }
+ 
+
+  static std::string description(){
+    return
+      "The comment input gives the new comment cards that are to be added\n"
+      "to the section MF=1/MT=451 on the output ENDF file.\n"
+      "\n"
+      "The comment cards are terminated by an empty card.";
+  }
+  
+  static Value_t defaultValue() { return nullopt; }
+
+  static bool verify( Value_t str ){ return str->length() <= 80; } 
```

Comment:
The length of these comments must be <= 66 characters. This comment becomes part of a [TEXT Record](https://github.com/njoy/ENDFtk/blob/master/src/ENDFtk/TextRecord.hpp)